.devcontainer/images/.claude/RTK.md

```
@@ -0,0 +1,69 @@
+rtk gain --graph      # ASCII graph (last 30 days)
+rtk gain --daily      # Day-by-day breakdown
+rtk discover          # Analyze Claude Code history for missed opportunities
+rtk proxy <cmd>       # Execute raw command without filtering (for debugging)
+```
+
+## How It Works
+
+RTK installs as a PreToolUse hook in Claude Code. It transparently rewrites
+Bash commands before execution to compress their output:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)